domain-server/src/DomainServer.h

```diff
@@ -228,6 +228,7 @@ private slots:
     bool processPendingContent(HTTPConnection* connection, QString itemName, QString filename, QByteArray dataChunk);
 
     bool forwardMetaverseAPIRequest(HTTPConnection* connection,
+                                    const QUrl& requestUrl,
                                     const QString& metaversePath,
```

Comment:
Perhaps add a comment on why both `requestUrl` and `metaversePath` are needed / how they differ - or add Doxygen for this method explaining all the parameters while you're here?

Reply:
Added Doxygen for it.